seyik/app.js

```diff
@@ -111,6 +110,58 @@ app.post("/posts/register", async (req, res) => {
   res.status(201).json({ message: "postCreated" });
 });
 
+app.patch("/posts", async (req, res) => {
+  const { postingContent, user_id } = req.body;
+
+  await appDataSource.query(
+    `UPDATE posts
+    SET
+    content = ?
+    WHERE posts.user_id = ? 
+    `,
+    [postingContent, user_id]
```

Comment:
```suggestion
    [postingContent, userId]
```
위에서 camelCase로 수정함에 따라 맞춰서 바꿔주세요